tests/_support/AcceptanceTester.php

```diff
@@ -18,9 +18,45 @@
 */
 class AcceptanceTester extends \Codeception\Actor
 {
-    use _generated\AcceptanceTesterActions;
+	use _generated\AcceptanceTesterActions;
 
-    /**
-     * Define custom actions here
-     */
+	/**
+	 * Custom actions here
+	 */
+
+	/**
+	 * Trigger actions during remote acceptance test executions.
+	 *
+	 * @param string $file File path.
+	 * @param callable $function Reference to a static class method.
+	 * @param bool $rollback_on_shutdown Rollback changes on shutdown.
+	 *
+	 * @throws Exception Throws an exception if passed parameters are not as expected.
+	 *
+	 * @return void
+	 */
+	public function preLoadConfig( string $file, callable $function, bool $rollback_on_shutdown = true ) {
+		$path = 'tests/include.php';
```

Comment:
This is a relative path; should be made into an absolute one, otherwise it will depend on the current working directory and have strange behaviour on execution.

Reply:
Yeah, although tests are expected to always run in the root directory, but it's better to be explicit.